src/platform/plugins/shared/dashboard/server/content_management/v3/types.ts

```diff
@@ -32,7 +32,9 @@ import {
   optionsSchema,
 } from './cm_services';
 import { CONTENT_ID } from '../../../common/content_management';
-import { DashboardSavedObjectAttributes } from '../../dashboard_saved_object';
+import { type DashboardSavedObjectAttributes } from '../../dashboard_saved_object';
+
+export { type DashboardSavedObjectAttributes } from '../../dashboard_saved_object';
```

Comment:
@dominiqueclarke Duplication with above, right?

Reply:
One is an import and one is an export

Reply:
Oops you are right!